interface/src/main.cpp

```diff
@@ -218,7 +218,7 @@ int main(int argc, const char* argv[]) {
     }
     qDebug() << "UserActivityLogger is enabled:" << ual.isEnabled();
 
-    if (ual.isEnabled()) {
+    if (ual.isCrashEnabled()) {
```

Comment:
perhaps rename to `isCrashReportingEnabled()`    , crashes can always happen even if not reported